pilot/pkg/proxy/envoy/v2/ads.go

```diff
@@ -342,7 +342,7 @@ func (s *DiscoveryServer) StreamAggregatedResources(stream ads.AggregatedDiscove
 
 				// Already got a list of endpoints to watch and it is the same as the request, this is an ack
 				if listEqualUnordered(con.Clusters, clusters) {
-					adsLog.Debugf("ADS:EDS: ACK %s %s (%s) %s %s", peerAddr, con.ConID, con.node.ID, discReq.VersionInfo, discReq.ResponseNonce)
+					adsLog.Debugf("ADS:EDS: ACK %s %s (%s) %s %s", peerAddr, con.ConID, discReq.VersionInfo, discReq.ResponseNonce)
```

Comment:
didn't update the format string it looks like

Reply:
good catch, the lint failed